notes/symenc.tex

```diff
@@ -59,6 +170,21 @@ \section{Randomized and Nonce-Respecting Symmetric Encryption}
 Return $\Enc(M_b)$
 }
 
+Notice that that $\advB$ playing the $\RORreal$ game is equivalent to 
+$\advA$ playing the $\INDCPA$ game, such that
+
+\bnm
+\Prob{\RORreal_{\SEscheme}^{\advB} \Rightarrow 1} =
+\Prob{\INDCPA_{\SEscheme}^{\advA} \Rightarrow \true}
+\enm
+
+Also notice that for $\advB$ playing the $\RORrand$ game, its oracle returns
+random bitstrings instead of actual ciphertext such that $\advA$, used by
+$\advB$, gets no information about the scheme whatsoever.  Thus
```

Comment:
ah, ok... my intuition in the first comment here is totally wrong, ignore it.  I've re-reviewed Bellare-Rogaway and what matters is that the _output of the oracle_ given to the adversary looks random (the similar intuition in the proof of Lemma 5.16 being an example).  So your original text is correct.  Basically, as Bellare-Rogaway says, "This tells us that any given output sequence from the oracle is equally likely in both worlds. Since the adversary determines its output bit based on this output sequence, its probability of returning 1 must be the same in both worlds".  This intuition wasn't clear to me from the original text, so while the text is actually right, I still might recommend tweaking.

Perhaps something like adding a sentence before "Thus," saying "Because these strings are random and independently generated, any sequence of oracle outputs is equally likely to occur regardless of the value of b. Because the output of A is determined by the choice of oracle output, this means A is equally likely to return 1 for both values of b."